### ISSUE TITLE
Unable to use on NVDA

### ISSUE THREAD

**Issue by TheQuinbox** [RHVoice/RHVoice#483]
Hi,
I attempted to install this addon to use with the NVDA screen reader, and it installed fine. But when attempting to select it in the synth dialog, I get an error. The following is written to the log:
```
ERROR - synthDriverHandler.setSynth (12:52:27.975) - MainThread (14048):
setSynth failed for RHVoice
Traceback (most recent call last):
  File "synthDriverHandler.pyc", line 462, in setSynth
  File "synthDriverHandler.pyc", line 428, in getSynthInstance
  File "C:\Users\quin\AppData\Roaming\nvda\addons\RHVoice\synthDrivers\RHVoice\__init__.py", line 791, in __init__
    raise RuntimeError("RHVoice: initialization error")
RuntimeError: RHVoice: initialization error
INFO - synthDriverHandler.setSynth (12:52:27.976) - MainThread (14048):
Falling back to previous synthDriver ibmeci
```
NVDA version 2021.3.3, Windows 10 64-bit.
Any ideas? I've seen a few of my friends install and use it with no issue, although I don't know what version of NVDA they were using.

**Comment by zstanecic** [comment]
Hi!

Have you installed any voice?

Driver with addon is insufficient.

**Comment by TheQuinbox** [comment]
Yeah, I just saw that. I feel kind of stupid now, although that fact should probably be made a bit clearer on the binaries page, where I got this addon from.
